lib/saluki-core/src/topology/graph.rs

```diff
@@ -447,9 +447,9 @@ mod test {
     fn invalid_input_id() {
         let mut graph = Graph::default();
         let result = graph
-            .with_source("in_log", DataType::Log)
-            .with_destination("out_log", DataType::Log)
-            .with_edge_fallible("in log", "out_log")
+            .with_source("in_eventd", DataType::EventD)
```

Comment:
This, and all changes in this file, are just renaming to reflect the fact I removed `DataType::Log` and `DataType::Trace`.